src/Ormolu/Processing/Cpp.hs

```diff
@@ -60,3 +61,14 @@ cppLines input = IntSet.fromAscList $ go Outside (T.lines input `zip` [1 ..])
             inConditional = case state of
               InConditional {} -> True
               _ -> False
+
+-- | Replace all lines affected by CPP with blank lines.
+eraseCppLines :: Text -> Text
+eraseCppLines input =
+  T.intercalate "\n" . fmap eraseCpp $ T.lines input `zip` [1 ..]
```

Comment:
Nit: What about `T.unlines` instead of `T.intercalate "\n"` for symmetry/consistency?

Reply:
Good idea, done!